### PR TITLE
Added a warning until we can get the runtime audited

### PR DESCRIPTION
**Pull request by DavidM-D**

### CODE REVIEW

**Review by idea404 — COMMENTED** [review]
LGTM. 

Might be worth mentioning that the first major release version will be production ready.

**Review by volovyks — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+> **Warning**
+> This is alpha software. The JavaScript runtime has not been audited yet, use [Rust](https://github.com/near/near-sdk-rs) if your contract holds value.
```
Comment (thisisjoshford) [inline README.md]:
```suggestion
> **Warning**
> 
> This SDK is currently in **[`Alpha`](https://github.com/near/near-sdk-js/releases/)**. 
> 
> The JavaScript runtime has not been fully audited. For creating smart contracts that hold value please use [`near-sdk-rs`](https://github.com/near/near-sdk-rs). 
> 
> Help contribute!
> - [Report issues you encounter](https://github.com/near/near-sdk-js/issues) 🐞 
> - [Provide suggestions or feedback](https://github.com/near/near-sdk-js/discussions) 💡 
> - [Show us what you've built!](https://github.com/near/near-sdk-js/discussions/categories/show-and-tell) 💪  
```

**Review by thisisjoshford — APPROVED** [review]
Added some minor suggestions. Thanks @DavidM-D!

**Review by ailisp — APPROVED** [review]
Looks good!

**Pull request merged**